docs/environment.yml

```diff
@@ -1,4 +1,3 @@
-name: clinica_docs
 channels:
   - defaults
   - conda-forge
```

Comment:
Now that RTD does not use Conda, we should probably get rid of this file.

Reply:
Done!